system/Autoloader/Autoloader.php

```diff
@@ -210,7 +191,24 @@ public function removeNamespace(string $namespace)
 		return $this;
 	}
 
-	//--------------------------------------------------------------------
+	/**
+	 * Load a class using available class mapping.
+	 *
+	 * @param string $class
+	 *
+	 * @return string|false
+	 */
+	public function loadClassmap(string $class)
+	{
+		$file = $this->classmap[$class] ?? '';
+
+		if (is_string($file) && $file !== '')
```

Comment:
This seems like a whole lot of effort not to use `empty()`.

Reply:
I'm using `phpstan/phpstan-strict-rules` in my workflows a lot, and one of the strict rules I'm following is the disallowance of `empty()` which implies loose comparison.

Reply:
Understood. I'm not against this verbose, strict approach but I personally prefer `empty()` when it saves multiple conditions.